test/sample-spec.js

```diff
@@ -0,0 +1,5 @@
+describe("Sample test",function(){
```

Comment:
Does this conform?
describe("Sample test", function() {
  it('This is a sample test', function() {
    expect(true).toEqual(true);
  });
});